pegasus/src/social_metadata.rb

```diff
@@ -0,0 +1,155 @@
+# Centralized social metadata for a few key pages:
+#   code.org/
+#   csedweek.org/
+#   hourofcode.com/
+#   code.org/challenge
+
+def get_social_metadata_for_page(request)
+  # Important:
+  #   - image should always come before video
+  #   - description should always come before description_twitter
+  social_tags = {
+    "code.org" => {
+      "soon-hoc" => {
+        title: I18n.t(:og_title),
+        description: I18n.t(:og_description),
+        image: {path: "/images/fit-1220/social-media/default-og-image.jpg", width: 1220, height: 640}
+      },
+      "week-before-hoc" => {
+        title: I18n.t(:og_title_soon),
+        description: I18n.t(:og_description_celeb),
+        image: {path: "/images/fit-1220/social-media/celeb-challenge.jpg", width: 1220, height: 640}
+      },
+      "actual-hoc" => {
+        title: I18n.t(:og_title_here),
+        description: I18n.t(:og_description_celeb),
+        image: {path: "/images/fit-1220/social-media/celeb-challenge.jpg", width: 1220, height: 640}
+      },
+      "default" => {
```

Comment:
Are there specific tags for post-hoc?

Reply:
Good catch.  Added a fix so that post-hoc looks like default.